Spurious error when using --isolatedModules in development

https://github.com/typed-ember/glint/blob/e35714c5fbccf279eaf476dd6e0b2161397085d2/packages/environment-ember-loose/-private/index.ts

This file needs `export {};` to avoid a spurious error when typechecking a project that uses `--isolatedModules`.

```
../../../repos/glint/packages/environment-ember-loose/-private/index.ts:1:1 -
error TS1208: 'index.ts' cannot be compiled under '--isolatedModules' because it is considered a global
script file. Add an import, export, or an empty 'export {}' statement to make it a module.
```

This issue only affects use of a directly-linked glint repo (e.g. during development / testing of glint itself using a different project as the subject). The generated `index.d.ts` does contain the `export {};`, so users of the npm package are unaffected.

--- a/packages/environment-ember-loose/-private/index.ts
+++ b/packages/environment-ember-loose/-private/index.ts
@@ -4,3 +4,4 @@
 
 /// <reference path="../registry/index.d.ts" />
 /// <reference path="./dsl/integration-declarations.d.ts" />
+export {};

--- a/packages/environment-ember-template-imports/-private/index.ts
+++ b/packages/environment-ember-template-imports/-private/index.ts
@@ -3,3 +3,4 @@
 // `import '@glint/environment-ember-template-imports'` somewhere in their project.
 
 /// <reference path="./dsl/integration-declarations.d.ts" />
+export {};

--- a/packages/environment-glimmerx/-private/index.ts
+++ b/packages/environment-glimmerx/-private/index.ts
@@ -4,3 +4,4 @@
 
 /// <reference path="../globals/index.d.ts" />
 /// <reference path="./dsl/integration-declarations.d.ts" />
+export {};

